docs/ibc/light-clients/localhost/overview.md

```diff
@@ -0,0 +1,41 @@
+<!--
+order: 1
+-->
+
+# `09-localhost`
+
+## Overview
+
+Learn about the 09-localhost light client module. {synopsis}
+
+The 09-localhost light client module implements a localhost loopback client with the ability to send and receive IBC packets to and from the same state machine.
+
+### Context
+
+In a multichain environment, application developers will be used to developing cross-chain applications through IBC. From their point of view, whether or not they are interacting with multiple modules on the same chain or on different chains should not matter. The localhost client module enables a unified interface to interact with different applications on a single chain, using the familiar IBC application layer semantics.
+
+### Implementation
+
+There exists a [single sentinel `ClientState`](./client-state.md) instance with the client identifier `09-localhost`.
+
+To supplement this, a [sentinel `ConnectionEnd` is stored in core IBC](./connection.md) state with the connection identifier `connection-localhost`. This enables IBC applications to create channels directly on top of the sentinel connection which leverage the 09-localhost loopback functionality.
+
+[State verification](./state-verification.md) for channel state in handshakes or processing packets is reduced in complexity, the `09-localhost` client can simply compare bytes stored under the standardized key paths.
+
+### Localhost vs *regular* client
+
+The localhost client aims to provide a unified approach to interacting with applications on a single chain, as the IBC application layer provides for cross-chain interactions. To achieve this unified interface though, there are a number of differences under the hood compared to a 'regular' IBC client (excluding `06-solomachine` and `09-localhost` itself).
+
+The table below lists some important differences:
+
+|  | Regular client | Localhost |
+| - | -------------- | --------- |
+| Number of clients | many instances of a client *type* corresponding to different counterparties | 1 single sentinel client with the client identifier `09-localhost`|
+| Client support | governance gated access to `allowed_clients` | added to `allowed_clients` by default |
+| Client creation | Relayer (permissionless) | the 09-localhost `ClientState` is instantiated in the `InitGenesis` handler of the 02-client submodule in core IBC |
+| Number of connection(s) | many connections, 1 (or more) per client | single sentinel `ConnectionEnd` representing a connection |
+| Connection creation | connection handshake, provided underlying client | the `ConnectionEnd` is created and set in store via the `InitGenesis` handler of the 03-connection submodule in core IBC |
+| Counterparty | underlying client, representing another chain | `09-localhost` |
+| Channel creation | channel handshake, provided underlying connection | channel handshake, **out of the box** |
```

Comment:
What does this mean? It seems a bit "hype" to me. "out of the box" isn't very impressive in this situation since it's just talking about a connection existing. It takes less than 30 seconds to initiate a connection using "regular clients". I think I'd prefer to minimize the amount of information that has to be read and assume the reader is able to deduce that channels can be immediately created using the sentinel localhost connection

Reply:
I've removed the "Channel Creation" row as channels are more or less treated the same

Reply:
Didn't intend to be claiming it was an 'impressive' feat by any means :) My thinking here was "what does all of this mean if I'm a dev who wants to start testing with localhost when used to a setup having 2 chains"? And then the only thing changing really from that POV is that I can immediately create my channel vs first having to create client and connection... so more pragmatic than hype 

Sure the reader could deduct it, but that holds for everything in the table. I personally just like when I can quickly gather all info from an overview table when I need a quick reminder

Reply:
That makes sense. Apologies if my comment seemed strong, that wasn't the intention. I'm just looking to make sure the language remains neutral. 

I have no preference if we keep this section but maybe we could use the wording "channel handshake on top of existing connection end `connection-localhost`" or something like that

Reply:
no worries, I learned now you don't like hype trains lol 🚂